int8 stop_flag
int8 init_flag
int8 control_mode
float64 ref_pos
float64 ref_vel
float64 kp
float64 kd